src/peerbook/libp2p_peer.erl

```diff
@@ -8,7 +8,8 @@
                        connected => [binary()],
                        nat_type => nat_type(),
                        network_id => binary(),
-                       associations => association_map()
+                       associations => association_map(),
+                       timestamp => non_neg_integer()
```

Comment:
Why is the timestamp for a peer here? The stale time is only for the peer of _this_ swarm, meaning the peerbook manages it

Reply:
This was before I understood what timestamp and stale_time were doing